ooiui/templates/c2/missionLoad.html

```diff
@@ -160,32 +169,72 @@
     });
 
     this.listenTo(this, 'missionTable:rowSelected', function(model) {
-      console.log('selected');
-
       var scriptlines = [];
 
       _.each(model.get('mission'),function(scriptline){
         scriptlines.push(scriptline);
       });
       $('#missionScript').html("<pre class='pre-scrollable mission-code'>"+scriptlines.join("<br>")+"</pre>");
+      $('#missionDelete').html("<button id='missionDelete' value='"+model.get('mission_id')+"' type='button' class='btn btn-sm btn-default'>Delete Selected Mission</button>");
+      $('#missionUpLoadNow').remove();
+      $('#missionUpLoad').attr('disabled', true);
+      document.getElementById("missionLoad").value='';
```

Comment:
No there isn't a model. There is just a file being pulled in by the user.  Do you have any other concerns on this JQuery? because i don't really understand your concern.